packages/common/src/ui/notifications/AlertModal.tsx

```diff
@@ -5,7 +5,7 @@ import { AlertIcon } from '@common/icons';
 import { BasicModal } from '@common/components';
 
 export interface AlertModalProps {
-  message: string;
+  message: React.ReactNode;
```

Comment:
just so I could use a `<ul/>` in the message. They can't be a child of `<p>` which the `<Typography/>` uses by default. Could specify a different component for that.. but since there is a div rendered anyway and it doesn't affect the look, I've just removed the `<Typography/>`